streams/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java

```diff
@@ -155,4 +163,20 @@ public void createTopic(final String topic,
     public void deleteTopic(final String topic) {
         brokers[0].deleteTopic(topic);
     }
+
+    public void waitForTopics(final StreamsConfig config, final String...topics) throws InterruptedException, IOException {
+        final StreamsKafkaClient streamsKafkaClient = new StreamsKafkaClient(config);
```

Comment:
Hmm, can't add it the the java `TestUtils` class as clients doesn't have a dependency on core.
Thought i'd add it to `StreamsTestUtils`, but can't as it breaks checkstyle:
```
[ant:checkstyle] /Users/damian/dev/apache/kafka/streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java:18:1: Disallowed import - kafka.api.PartitionStateInfo.
[ant:checkstyle] /Users/damian/dev/apache/kafka/streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java:19:1: Disallowed import - kafka.api.Request.
[ant:checkstyle] /Users/damian/dev/apache/kafka/streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java:20:1: Disallowed import - kafka.server.KafkaServer.
[ant:checkstyle] /Users/damian/dev/apache/kafka/streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java:21:1: Disallowed import - kafka.server.MetadataCache.
[ant:checkstyle] /Users/damian/dev/apache/kafka/streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java:26:1: Disallowed import - scala.Option.
```